src/gps/gps.c

```diff
@@ -177,6 +201,7 @@ void GPS_sample_update(GpsSample *newSample)
     updateFullDateTime(newSample);
     g_gpsSnapshot.deltaFirstFix = newSample->time - g_timeFirstFix;
     g_gpsSnapshot.previousPoint = prevPoint;
+    g_gpsSnapshot.bearing = gps_bearing(&prevPoint, &(newSample->point));
```

Comment:
I'm not convinced that am EMA filter is the right solution here as the values can change quickly and we don't want any delay in there.  This sort of filter would be much better suited for our gas gauge sender to smooth it out (as it really really needs it).  Perhaps a minimum speed or minimum distance would be good here.  I'm leaning towards the latter since that way we can get gps bearing at lower speeds with minimal extra memory usage.